src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -122,6 +122,9 @@ public static Builder builder(Class<?> type) {
   public Builder toBuilder() {
     Builder builder = new Builder(type);
     builder.members.putAll(members);
+    for (Map.Entry<String, List<CodeBlock>> entry : builder.members.entrySet()) {
```

Comment:
...or it resolves to a code block like:
```java
  public Builder toBuilder() {
    Builder builder = new Builder(type);
    builder.members.putAll(Util.mutableMultimap(members));
    return builder;
  }
```
Here, there's a map populated twice. I'm still favoring the inline method as initially proposed.